Plugins/NativeCamera/Source/Apple/NativeCameraImpl.h

```diff
@@ -15,7 +15,7 @@ namespace Babylon::Plugins
         
         Impl(Napi::Env env, bool overrideCameraTexture);
         ~Impl();
-        arcana::task<void, std::exception_ptr> Open(uint32_t width, uint32_t height, bool frontCamera);
+        arcana::task<const Camera::CameraDimensions*, std::exception_ptr> Open(uint32_t maxWidth, uint32_t maxHeight, bool frontCamera);
```

Comment:
Why pass this as a pointer? It's a tiny struct, and there is no intention for the caller to be able to observe mutations to this struct right? Seems like it should just be returned by value.